yb-voyager/src/datafile/csvdatafile.go

```diff
@@ -30,29 +31,29 @@ func (df *CsvDataFile) SkipLines(numLines int64) error {
 }
 
 func (df *CsvDataFile) NextLine() (string, error) {
-	/*
-		// TODO: resolve the issue in counting bytes with csvreader
-		// here csv reader can be more useful in case filter table's columns
-		// currently using normal file reader will also work
-		fields, err := df.reader.Read()
+	var line string
+	var err error
+	for {
+		startOffset := df.reader.InputOffset()
+		// read a record from csv file
+		_, err := df.reader.Read()
 		if err != nil {
 			return "", err
```

Comment:
Add context around the err:
```
return "", fmt.Errorf("read record from CSV file %q: %w", df.file.Name(), err)
```

Reply:
can't do that formatting here, we need the `EOF` in exactly the same form in the caller.

Reply:
The caller should use `errors.Is(err, io.EOF)` to check the error.